src/.vuepress/public/editor.html

```diff
@@ -103,6 +102,16 @@
       background-image: url('data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCA0OCA0OCI+PHBhdGggZmlsbD0iI2RkZCIgZD0iTTE2IDEwdjI4bDIyLTE0eiIvPjwvc3ZnPg==')
     }
 
+    @media only screen and (max-width: 700px) {
+      #tabs .tab span {
+        visibility: collapse;
```

Comment:
Isn't `collapse` specifically for tables or something? Are there problems with `display: none`?

Reply:
Unfortunately with `display: none` it looks like this:
<img width="377" alt="Снимок экрана 2022-07-01 в 00 20 58" src="https://user-images.githubusercontent.com/1301959/176780353-d4bef8da-329a-4131-8f16-ff80118573a7.png">

Reply:
`display: none` was the obvious choice in the beginning, but then had to do otherwise

Reply:
I see. Well, I guess since it's not a table, it can just be `visibility: hidden`? Other than that, on my system, there isn't a 1px divider between the tab and the editor contents - is it possible to make it the tabs one pixel higher maybe?

Reply:
> Other than that, on my system, there isn't a 1px divider between the tab and the editor contents

It only happening in debug mode with console and layout ruler. I don't think it should be a problem in normal mode so don't worry about this gap